spec/app/domain/authentication/webservice_spec.rb

```diff
@@ -1,11 +1,4 @@
-# frozen_string_literal: true
-
-require 'authentication/webservice'
-
-require 'rspec'
-$: << '../../../../app/domain'
-require 'authentication/webservice'
-
+require 'spec_helper'
```

Comment:
This is one of the `spec_helper` replacements I made. Did you want to change it back?

Reply:
If it's working, let's leave it.  

All this stuff should be revisited in a larger project about how to reorganize rspec, which I already have notes for somewhere in our dev notes repo.